registrace_bp.py

```diff
@@ -1,73 +1,80 @@
 from flask import Blueprint, render_template, request, redirect, url_for, flash, abort, current_app
-# from wtforms import Form, BooleanField, StringField, PasswordField, validators
+from wtforms import Form, BooleanField, StringField, PasswordField, IntegerField, validators
 from flask_login import current_user
 import db_funkce
 
 
+class RegistrationForm(Form):
+    username = StringField(
+        'Jméno*:', [validators.Required(message='Bez jména to nepůjde.')])
+    surname = StringField(
+        'Příjmení*:', [validators.Required(message='Bez příjmení to nepůjde.')])
+    street = StringField('Ulice,č.p./č.o.:')
+    city = StringField(
+        'Město/Obec*:', [validators.Required(message='Vyplň město, kde bydlíš.')])
+    postcode = IntegerField('PSČ*:', [validators.Required(message='Bez PSČ to nepůjde.'),
+                                      validators.Length(min=5, max=5, message='PSČ zadej bez mezer.')])
+    email = StringField('E-mail*:', [validators.Required(message='Bez mailu to nepůjde.', ),
+                                     validators.Email(message='Chybný tvar emailové adresy.'), validators.Length(min=4, max=40)])
+    phone = IntegerField('Telefon*:', [validators.Required(message='Bez telefonního čísla to nepůjde.'), validators.Length(
+        min=11, max=11, message='Telefonní číslo zadej bez předvolby a ve tvaru xxx xxx xxx.')])
+    password = PasswordField('Heslo*:', [validators.Required(message='Bez hesla to nepůjde.'), validators.EqualTo(
+        'confirm_password', message='Hesla se musejí shodovat.'), validators.Length(min=8, max=20, message='Heslo musí být alespoň 6 znaků dlouhé.')])
```

Comment:
Horní limit na délku hesla bych nedával, ať lidi používají co chtějí - stejně to potom zahashujeme na 32 znaků...